### DISCUSSION TITLE
Publish to firebase failes becasue of contents in release_notes.txt

### DISCUSSION THREAD

**Discussion by SEGVeenstra** [codemagic-ci-cd/codemagic-docs#2205] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Add a `release_notes.txt` to the repository root
2. Publish app to firebase

### Expected results

The app to be published to firebase

### Actual results

```bash
> firebase appdistribution:distribute /Users/builder/clone/build/ios/ipa/scavenger_hunt.ipa --app <id> --groups phase_1_ios --release-notes '- Fix dark mode for dialogs
- Fix dark mode for checkboxes
- Remove buttons on Google map view
- Add trimming of text fields on save'
error: unknown option '-F'

Failed to publish /Users/builder/clone/build/ios/ipa/scavenger_hunt.ipa to Firebase App Distribution: error: unknown option '-F'
.

Build failed :|
```

It looks like it's not escaping the content correctly?

### Build id (optional)

655895976343b134287dd166

**Comment by himesh-cm** [comment]
Hi @SEGVeenstra, Please remove the `-` before `Fix` from your release notes and try re-running the build again.

**Reply by SEGVeenstra** [reply]
> Hi @SEGVeenstra, Please remove the `-` before `Fix` from your release notes and try re-running the build again.

Yeah I added a sentence above it but that's just a work around.

This should be fixed.

**Reply by himesh-cm** [reply]
Please use the above workaround, a permanent Fix is underway, I will tag this thread and keep you posted.

**Reply by himesh-cm** [reply, marked as answer]
Hi, this has been resolved. Please let me know if you face any further issues.
